doc/whats_new.rst

```diff
@@ -56,6 +56,10 @@ Enhancements
     generated during build. Distribution packages will still contain generated
     C/C++ files. By `Arthur Mensch`_
 
+    - Enhanced MLP with Adams, Nesterov's momentum, and early stopping (`#5214 <https://github.com/scikit-learn/scikit-learn/pull/5214>`_)
+
+    - Enhanced with colorblind compatibility to various examples by `Johanna Hansen`_
+
```

Comment:
Please also add these improvements:
    - Made scaling a sparse matrix along axis 0 to accept a CSR format by default by `Manoj Kumar`_

    - Improve tests checking sample_weight in LinearRegression and Ridge by `Giorgio Patrini`_

    - Added the learning curve documentation to model_selection.rst by `Raghav R V`_